api/server/httputils/httputils.go

```diff
@@ -6,14 +6,12 @@ import (
 	"net/http"
 	"strings"
 
+	"github.com/docker/docker/api/server/middleware"
 	"github.com/pkg/errors"
 	"github.com/sirupsen/logrus"
 	"golang.org/x/net/context"
 )
 
-// APIVersionKey is the client's requested API version.
-const APIVersionKey = "api-version"
```

Comment:
moved this to `middleware/version.go` as that's where this key is actually set, so it seemed more logical to define the name there 😅